autogen/dockerversion/dockerversion.go

```diff
@@ -0,0 +1,14 @@
+// Package dockerversion is auto-generated at build-time
+package dockerversion
+
+// Default build-time variable for library-import.
+// This file is overridden on build with build-time informations.
+const (
+	GITCOMMIT string = "library-import"
```

Comment:
Didn't want to make too much changes across the sources, that's why I kept these `CAPS` and used `const` (that way golint is happy).

I can make the change but I felt the change should be not too much intrusive (at first :stuck_out_tongue_closed_eyes:)